docs/scos/dev/glue-api-guides/202200.0/managing-wishlists/managing-wishlist-items.md

```diff
@@ -0,0 +1,258 @@
+
+### Response
+
+Resonse sample:
+
+```json
+{
+		"data": {
+			"type": "wishlist-items",
+			"id": "064_18404924",
+			"attributes": {
+				"sku": "064_18404924"
+			},
+			"links": {
+				"self": "https://glue.mysprykershop.com/wishlists/c917e65b-e8c3-5c8b-bec6-892529c64b30/wishlist-items/064_18404924"
+			}
+		}
+	}
+```
+<details><summary>Response sample: add a configurable product to a wishlist</summary>
+
+```json
+{
+    "data": {
+        "type": "wishlist-items",
+        "id": "093_24495843_98bf36f052d23f10a8a081694ad4f45e",
+        "attributes": {
+            "productOfferReference": null,
+            "merchantReference": "MER000001",
+            "id": "093_24495843_98bf36f052d23f10a8a081694ad4f45e",
+            "sku": "093_24495843",
+            "availability": {
+                "isNeverOutOfStock": false,
+                "availability": false,
+                "quantity": "0.0000000000"
+            },
+            "productConfigurationInstance": {
+                "displayData": "{\"Preferred time of the day\": \"Afternoon\", \"Date\": \"9.12.2021\"}",
+                "configuration": "{\"time_of_day\": \"2\"}",
+                "configuratorKey": "DATE_TIME_CONFIGURATOR",
+                "isComplete": true,
+                "quantity": 3,
+                "availableQuantity": 4,
+                "prices": [
+                    {
+                        "netAmount": 23434,
+                        "grossAmount": 42502,
+                        "priceTypeName": "DEFAULT",
+                        "volumeQuantity": null,
+                        "currency": {
+                            "code": "EUR",
+                            "name": "Euro",
+                            "symbol": "€"
+                        },
+                        "volumePrices": [
+                            {
+                                "grossAmount": 165,
+                                "netAmount": 150,
+                                "quantity": 5
+                            },
+                            {
+                                "grossAmount": 158,
+                                "netAmount": 145,
+                                "quantity": 10
+                            },
+                            {
+                                "grossAmount": 152,
+                                "netAmount": 140,
+                                "quantity": 20
+                            }
+                        ]
+                    }
+                ]
+            },
+            "prices": []
+        },
+        "links": {
+            "self": "http://glue.de.spryker.local/wishlists/b70e1073-a740-5a48-bb5e-0449a9e51d53/wishlist-items/093_24495843_98bf36f052d23f10a8a081694ad4f45e"
+        }
+    }
+}
+```
+
+</details>
+
+<a name="wishlishlist-items-response-attributes"></a>
```

Comment:
Is it normal to have a link without an HREF tag?

Reply:
this is an anchor to which I'm referring to from another place in the document